src/toil/utils/toilStatus.py

```diff
@@ -219,6 +223,23 @@ def getStatus(jobStoreName: str) -> str:
                 pass
         return 'RUNNING'
 
+
+    def readBusMessages(self) -> None:
```

Comment:
This should be `snake_case`.

Reply:
Also all the other function names start with "print" and this one starts with "read" but actually it prints anyway.